src/pages/iou/MoneyRequestSelectorPage.js

```diff
@@ -96,6 +101,31 @@ function MoneyRequestSelectorPage(props) {
 
     const [isDraggingOver, setIsDraggingOver] = useState(false);
 
+    const RenderMoneyRequestAmountPage = useCallback(
```

Comment:
Note that I just did this as a quick workaround to make sure these components got the props they needed. In a non-POC implementation we could just reference the components directly and each component could subscribe to the data it needs.